code/game/gamemodes/clock_cult/clock_items/clock_components.dm

```diff
@@ -11,8 +11,14 @@
 
 /obj/item/clockwork/component/pickup(mob/living/user)
 	..()
-	if(iscultist(user) || (user.mind && user.mind.assigned_role == "Chaplain"))
+	if(iscultist(user) || (user.mind && user.mind.isholy))
 		user << "<span class='[message_span]'>[cultist_message]</span>"
+		if(user.mind && user.mind.isholy)
+			user << "<span class='boldannounce'>[SSreligion.Bible_deity_name]'s voice thunders: BEGONE FALSE THING!</span>"
```

Comment:
Pretty much all of what you get from cultists is slabs, brass sheets, and alloy shards, not components.

Also, I take some issue with the fact you're forcing a voice on what's basically a player-named-and-fluffed character, which is, pretty generally, a Bad Thing to do.